client/src/main/scala/org/apache/celeborn/client/LifecycleManager.scala

```diff
@@ -502,6 +505,12 @@ class LifecycleManager(appId: String, val conf: CelebornConf) extends RpcEndpoin
       return
     }
 
+    if (getPartitionType(shuffleId) == PartitionType.MAP) {
+      logError(s"[handleRevive] shuffle $shuffleId revived filed, because map partition don't support revive!")
```

Comment:
We really log an error message here? A comment is ok?

Reply:
Currently better log error message in lifecycleManager for tracing error event.